__tests__/fixtures/on-content/next.config.js

```diff
@@ -0,0 +1,9 @@
+const withMdxEnhanced = require('../../..')
+
+module.exports = withMdxEnhanced({
+  onContent: mdxContent => processContent(mdxContent)
+})()
+
+function processContent(content) {
+  console.log(content)
+}
```

Comment:
Can this file be removed?

Reply:
Removing this actually makes it less convenient when you want to run site within the fixture, I think there's some utility to having this separate from the `next.config-mock.js` which is used in the integration tests. 

That said, if there's a cleaner path, I'm happy to implement.